### PR TITLE
Moved DimArtOutcome to the static Dimensions folder. Also,removed the…

### PR DESCRIPTION
**Pull request by DennisGibz**
… where clause on the LineList fact Art under the reporting layer

### CODE REVIEW

File: Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql
Comment (nobert-mumo) [inline Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:1]: @DennisGibz maybe we can tidy up the Dim to get rid of the incremental upserts just to be explicit ?

Comment (DennisGibz) [reply Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:1]: @nobert-mumo either way I don't think that the dim under static dimension folder will ever be used again.

Wanted to maintain its structure from the originating folder.

Comment (nobert-mumo) [reply Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:1]: I agree the minimal is small indeed but, I think there is a use case for the script We might need to add new valid ARTOutcomes to the dim in future

Comment (DennisGibz) [reply Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:1]: @nobert-mumo above has now been factored in.

Comment (nobert-mumo) [reply Scripts/NDWH/ALL DIMENSIONS/STATIC DIMENSIONS/load_DimARTOutcome.sql:1]: @DennisGibz  sure looks good to me. We can look to uppercase the ARTOutcomeDescriptions , but should be fine

**Review by nobert-mumo — APPROVED** [review]

**Comment by DennisGibz** [comment]
This is now well catered for.

**Pull request merged**